### PR TITLE
Add forgot password for login

### PR DESCRIPTION
**Pull request by lhuynh**
- added initial project setup
- added appsettings.json for diff environment
- added forgot password logic for Auth0 db call
- added forgot password UI

### CODE REVIEW

File: AuthenticationFE/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
```
Comment (tgobena) [inline AuthenticationFE/Controllers/HomeController.cs]: Remove this controller if it is not used

File: AuthenticationFE/Models/User.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (tgobena) [inline AuthenticationFE/Models/User.cs]: To centralize the code you can use the username that we have in the WatchGuardIDCommon.WGIDModel project.   you can add the code you want in there and after check in you can get the data from nuget package

**Comment by tgobena** [comment]
In general, it could be now or latter see if we need all the auto-generated files.

File: AuthenticationFE/Controllers/HomeController.cs (hunk 1 above)
Comment (lhuynh) [inline AuthenticationFE/Controllers/HomeController.cs]: If I delete this how should I handle if a user navigates to the root? I currently have the home page populated with the forgot password link.

**Comment by lhuynh** [comment]
@tgobena sounds good, but can you give an example of what you mean by auto-generated files?

**Pull request merged**